app/views/merchants/index.erb

```diff
@@ -6,7 +6,7 @@
 
 <a href="/merchants/new">Create a merchant</a>
 <% @merchants.each do |merchant| %>
-<div class="merchant_<%= merchant.id %>">
```

Comment:
I kind of prefer this. or "merchant-<%etc....>

Reply:
Works for me.

Reply:
so this is now in master but I'll just keep it the way it is on my PR once i have merge conflicts. If I realized we could chat about it before I approved or anything I would have. Next time.